Userland/Libraries/LibWeb/HTML/HTMLInputElement.cpp

```diff
@@ -791,16 +791,39 @@ void HTMLInputElement::form_associated_element_was_inserted()
     create_shadow_tree_if_needed();
 }
 
+static bool is_in_same_radio_button_group(HTML::HTMLInputElement const& a, HTML::HTMLInputElement const& b)
```

Comment:
Can we get a link here to https://html.spec.whatwg.org/multipage/input.html#radio-button-group ?

Reply:
Yep! My bad!